### PR TITLE
Fix: Typos and Grammar errors

### PR DESCRIPTION
**Pull request by PixelTom**
Fixes a large number of typos and grammar issues.

Edit: in the process of splitting this PR into 1 file each. 

Craftworlds: https://github.com/BSData/wh40k/pull/8128
Drukhari: https://github.com/BSData/wh40k/pull/8129
Corsairs: https://github.com/BSData/wh40k/pull/8130
Harlequins: https://github.com/BSData/wh40k/pull/8131
Ynnari: https://github.com/BSData/wh40k/pull/8132
Chaos Knights: https://github.com/BSData/wh40k/pull/8133
Chaos Space Marines: https://github.com/BSData/wh40k/pull/8134
Chaos Daemons: https://github.com/BSData/wh40k/pull/8135
Dark Mechanium: Looks like this was removed from master so I've left my changes in the void.
Heretic Astartes: https://github.com/BSData/wh40k/pull/8136
Renegade and Heretics: https://github.com/BSData/wh40k/pull/8137
Thousand Sons: https://github.com/BSData/wh40k/pull/8138
Adepta Sororitas: https://github.com/BSData/wh40k/pull/8139
Adeptus Astra Telepathica As: https://github.com/BSData/wh40k/pull/8140
Custodes: https://github.com/BSData/wh40k/pull/8141
Adeptus Mechanicus: https://github.com/BSData/wh40k/pull/8142
Astra Militarum: https://github.com/BSData/wh40k/pull/8143
Black Templars: https://github.com/BSData/wh40k/pull/8144
Blood Angels: https://github.com/BSData/wh40k/pull/8145
Dark Angels: https://github.com/BSData/wh40k/pull/8146
Deathwatch: https://github.com/BSData/wh40k/pull/8147
Elucidian Starstriders: https://github.com/BSData/wh40k/pull/8148
FW Adeptus Astartes: https://github.com/BSData/wh40k/pull/8149
Death Korps: https://github.com/BSData/wh40k/pull/8150
Elysians: https://github.com/BSData/wh40k/pull/8151
Grey Knights: https://github.com/BSData/wh40k/pull/8152
Imperial Knights: https://github.com/BSData/wh40k/pull/8153
Inquisition: https://github.com/BSData/wh40k/pull/8154
Assassins: https://github.com/BSData/wh40k/pull/8155
Raven Guard: https://github.com/BSData/wh40k/pull/8156
Space Wolves: https://github.com/BSData/wh40k/pull/8157
Space Marines: https://github.com/BSData/wh40k/pull/8158
Ultramarines: https://github.com/BSData/wh40k/pull/8159
White Scars: https://github.com/BSData/wh40k/pull/8160
Necrons: https://github.com/BSData/wh40k/pull/8161
Orks: https://github.com/BSData/wh40k/pull/8162
General: https://github.com/BSData/wh40k/pull/8163
Genestealer Cultists: https://github.com/BSData/wh40k/pull/8164
Tyranids: https://github.com/BSData/wh40k/pull/8165
Tau: https://github.com/BSData/wh40k/pull/8166

### CODE REVIEW

File: Chaos - FW Renegade and Heretics.cat [hunk 1]
```diff
@@ -5598,7 +5598,7 @@
                     <characteristic name="S" typeId="59b1-319e-ec13-d466">10</characteristic>
                     <characteristic name="AP" typeId="75aa-a838-b675-6484">-3</characteristic>
                     <characteristic name="D" typeId="ae8a-3137-d65b-4ca7">D3</characteristic>
-                    <characteristic name="Abilities" typeId="837d-5e63-aeb7-1410">Reroll failed wound rolls on &lt;VEHICLE&gt;</characteristic>
```
Comment (amis92) [inline Chaos - FW Renegade and Heretics.cat]: It looks like casing got lost here - can you fix that?

File: Chaos - FW Renegade and Heretics.cat [hunk 2]
```diff
@@ -5692,7 +5692,7 @@
                     <characteristic name="S" typeId="59b1-319e-ec13-d466">User</characteristic>
                     <characteristic name="AP" typeId="75aa-a838-b675-6484">-1</characteristic>
                     <characteristic name="D" typeId="ae8a-3137-d65b-4ca7">D3</characteristic>
-                    <characteristic name="Abilities" typeId="837d-5e63-aeb7-1410">Reroll failed wound rolls on Non - &lt;VEHICLE&gt; models</characteristic>
```
Comment (amis92) [inline Chaos - FW Renegade and Heretics.cat]: It looks like casing got lost here - can you fix that?

File: Chaos - FW Renegade and Heretics.cat [hunk 3]
```diff
@@ -6252,12 +6252,12 @@
...
           </characteristics>
         </profile>
         <profile id="5f0b-0f18-4af0-6eb5" name="Artillery" hidden="false" typeId="72c5eafc-75bf-4ed9-b425-78009f1efe82" typeName="Abilities">
           <characteristics>
-            <characteristic name="Description" typeId="21befb24-fc85-4f52-a745-64b2e48f8228">An Eartshaker Carriage Battery can only fire its ranged weapon if a friendly &lt;RENEGADE&gt; guardsman crew model is within 3&quot;. A single Guardsman model cannot operate multiple Earthshaker Carriages in this way in a single turn, if all of the guardsmen crew within 6&quot; of an earthshaker Carriage are slain, it immediately shuts down and is removed from play.</characteristic>
+            <characteristic name="Description" typeId="21befb24-fc85-4f52-a745-64b2e48f8228">An earthshaker Carriage Battery can only fire its ranged weapon if a friendly &lt;RENEGADE&gt; guardsman crew model is within 3&quot;. A single Guardsman model cannot operate multiple Earthshaker Carriages in this way in a single turn, if all of the guardsmen crew within 6&quot; of an earthshaker Carriage are slain, it immediately shuts down and is removed from play.</characteristic>
```
Comment (amis92) [inline Chaos - FW Renegade and Heretics.cat]: It looks like casing got lost here - can you fix that?

File: Chaos - FW Renegade and Heretics.cat [hunk 4]
```diff
@@ -7035,7 +7035,7 @@
             <characteristic name="S" typeId="59b1-319e-ec13-d466">10</characteristic>
             <characteristic name="AP" typeId="75aa-a838-b675-6484">-3</characteristic>
             <characteristic name="D" typeId="ae8a-3137-d65b-4ca7">D3</characteristic>
-            <characteristic name="Abilities" typeId="837d-5e63-aeb7-1410">Reroll failed wound rolls on &lt;VEHICLE&gt;</characteristic>
+            <characteristic name="Abilities" typeId="837d-5e63-aeb7-1410">re-roll failed wound rolls on &lt;VEHICLE&gt;</characteristic>
```
Comment (amis92) [inline Chaos - FW Renegade and Heretics.cat]: It looks like casing got lost here - can you fix that?

File: Imperium - Astra Militarum - Library.cat [hunk 5]
```diff
@@ -3009,7 +3009,7 @@
             <characteristic name="Description" typeId="21befb24-fc85-4f52-a745-64b2e48f8228">During deployment, you can set this unit up underground instead of placing it on the battlefield. At the end of any of your Movement phases, this unit may drill up from the ground and into battle - set it up anywhere on the battlefield that is more than 9&quot; away from enemy models.</characteristic>
           </characteristics>
         </profile>
-        <profile id="1fc8-b024-e2a1-0dff" name="Seperate Orders" hidden="false" typeId="72c5eafc-75bf-4ed9-b425-78009f1efe82" typeName="Abilities">
+        <profile id="1fc8-b024-e2a1-0dff" name="separate Orders" hidden="false" typeId="72c5eafc-75bf-4ed9-b425-78009f1efe82" typeName="Abilities">
```
Comment (amis92) [inline Imperium - Astra Militarum - Library.cat]: It looks like casing got lost here - can you fix that?

File: Imperium - Grey Knights.cat [hunk 6]
```diff
@@ -8481,7 +8481,7 @@
       <characteristics>
         <characteristic name="Warp Charge" typeId="5ffd-b800-c317-532a">7</characteristic>
         <characteristic name="Range" typeId="fd64-cbc4-94de-24cc">6&quot;</characteristic>
-        <characteristic name="Details" typeId="ad96-dfa4-b4ed-656d">Ethereal Manipulation has a warp charge value of 7. If manifested, select one friendly GREY KNIGHTS unit within 6&quot; of this psyker. Unitl the start of your next Psychic phase, when resolving an attack made by a model in that unit in the Shooting phase a unit that is within half range, you can re-roll the hit roll.</characteristic>
+        <characteristic name="Details" typeId="ad96-dfa4-b4ed-656d">Ethereal Manipulation has a warp charge value of 7. If manifested, select one friendly GREY KNIGHTS unit within 6&quot; of this psyker. until the start of your next Psychic phase, when resolving an attack made by a model in that unit in the Shooting phase a unit that is within half range, you can re-roll the hit roll.</characteristic>
```
Comment (amis92) [inline Imperium - Grey Knights.cat]: It looks like casing got lost here - can you fix that?

**Review by amis92 — CHANGES_REQUESTED** [review]
It looks great with a few little mistakes.

**Comment by amis92** [comment]
@PixelTom We appreciate the changes.

This PR is huge however, and would benefit from being split into multiples.

Another reason is that this PR involves almost whole team to sign-off, and if you split the file changes into separate PRs it'd be much easier and faster to accept.

**Comment by PixelTom** [comment]
@amis92 thanks for the feedback! I can close this PR and make 1 for each file if you'd like. 

Edit: Doing that now! :)

**Comment by PixelTom** [comment]
Closing this. Will update description with the new PRs as I make them.

**Pull request closed without merging**